tests/unit/disco/components/TestAddon.js

```diff
@@ -289,20 +284,24 @@ describe(__filename, () => {
     });
 
     it('does render a logo for an extension', () => {
-      const root = renderAddon({ addon: result });
+      const { heading, ...addon } = result;
+      const root = renderAddon({ addon, heading });
```

Comment:
it's not clear to me why you have to pass in heading here since it's not part of the test.